x/clp/types/keys.go

```diff
@@ -35,16 +35,10 @@ var (
 	PmtpParamsPrefix                    = []byte{0x05} // Key to store the Pmtp params
 	RewardParamPrefix                   = []byte{0x06}
 	SymmetryThresholdPrefix             = []byte{0x07}
-	SwapAssetPermissionStorePrefix      = []byte{0x08}
 	LiquidityProtectionParamsPrefix     = []byte{0x09} // Key to store the Liquidity Protection params
 	LiquidityProtectionRateParamsPrefix = []byte{0x0a} // Key to store the Liquidity Protection rate params
```

Comment:
@joneskm I would suggest to fill the gap between `0x07` and `0x09` by decreasing two new prefixed by one

such as:

```
LiquidityProtectionParamsPrefix     = []byte{0x08} // Key to store the Liquidity Protection params
LiquidityProtectionRateParamsPrefix = []byte{0x09} // Key to store the Liquidity Protection rate params
```

Reply:
agreed, will push a change